src/components/common/BuyCryptoButton/index.tsx

```diff
@@ -21,7 +21,7 @@ const useOnrampAppUrl = (): string | undefined => {
 
 const useBuyCryptoHref = (): LinkProps['href'] | undefined => {
   const query = useSearchParams()
-  const safe = query.get('safe')
+  const safe = query?.get('safe')
```

Comment:
You seem to fix these randomly in various PRs. Is it really required? Can we fix _all_ of them in one separate PR?

Reply:
Whenever I run `git push` I get this and it causes the same lint errors. I am fixing all of them in each of my PRs and its always the same ones so once we have this merged to dev it should be resolved.

<img width="949" alt="Screenshot 2024-05-30 at 14 54 15" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/4127bffd-29d0-4694-a961-b64613f06029">